news/2021-09-26-project-loved-september-2021.md

```diff
@@ -0,0 +1,403 @@
+
+As usual, we would like to thank everyone who participated in the August votes. You're helping to keep this project alive by doing so, and we appreciate it! If you would like to view the results from the previous round, you may do so by clicking the following links:
+
+- [Results for osu!](https://osu.ppy.sh/community/forums/posts/8269959)
+- [Results for osu!taiko](https://osu.ppy.sh/community/forums/posts/8269958)
+- [Results for osu!catch](https://osu.ppy.sh/community/forums/posts/8269956)
+- [Results for osu!mania](https://osu.ppy.sh/community/forums/posts/8269955)
+
+September comes with good news for Project Loved! The [Project Loved website](https://loved.sh/) is now fully localisable into your native language. Much thanks to the contributors across the community, the website is available in Spanish, Finnish, Swedish and Turkish for the time being. Interested in translating the website to your native language? If so, head over to the [Localisation page of the Project Loved website](https://loved.sh/localization) and follow the steps there! If you wish to contribute to the Project Loved, feel free to visit the `#osu-loved` channel in the osu!dev Discord server. If website development is your thing, you can follow the ongoing development progress on [the GitHub repository](https://github.com/cl8n/project-loved-web).
+
+Before announcing the picks for this month, we would like to introduce you to the new faces of the Project Loved Team and wave goodbyes to those who recently parted ways with us. Already present with us as a metadata reviewer for the picks, [UberFazz](https://osu.ppy.sh/users/8646059), and [Doomsday](https://osu.ppy.sh/users/18983), a long-time osu! player and contributor, have rolled up their sleeves to assist other osu! captains with Loved nominations. Speaking of captains, we are also delighted to present you with [Gamelan4](https://osu.ppy.sh/users/9856910), our newest osu!taiko captain, tasked with picking your favourite dons and kats. Coming from the Global Moderation Team, [Zeus-](https://osu.ppy.sh/users/5464437) takes on the responsibility of beatmap content moderation alongside [Noffy](https://osu.ppy.sh/users/1541323) and [Pachiru](https://osu.ppy.sh/users/2850983). As for our last addition to the Project Loved Team, a renowned Beatmap Nominator, [Hivie](https://osu.ppy.sh/users/14102976) is here to make sure the metadata across all the nominations is accurate.
+
+Unfortunately, new arrivals tend to bring along several departures. We have recently parted ways with [X a v y](https://osu.ppy.sh/users/3738344) and [Alter-](https://osu.ppy.sh/users/4980256), osu!taiko and osu!mania captains respectively. Although they are no longer a part of the Project Loved Team, their contributions to the Loved scene are here to stay forever. We would like to thank them for their service and wish them luck in their future endeavours!
+Alright, enough said. Let's get on with this month's nominations:
```

Comment:
```suggestion
Unfortunately, new arrivals tend to bring along several departures. We have recently parted ways with [X a v y](https://osu.ppy.sh/users/3738344) and [Alter-](https://osu.ppy.sh/users/4980256), osu!taiko and osu!mania captains respectively. Although they are no longer a part of the Project Loved Team, their contributions to the Loved scene are here to stay forever. We would like to thank them for their service and wish them luck in their future endeavours!

Alright, enough said. Let's get on with this month's nominations.
```
new lines should have one empty line between separating them. this goes for all other occurrences like this